websites/D/DualFM/dist/metadata.json

```diff
@@ -3,12 +3,12 @@
     "name": "Jack Hansen",
     "id": "140590672657776641"
   },
-  "service": "DualFM",
+  "service": "Dual",
```

Comment:
service name and folder name must be the same. Note: if you change the service name rn everyone would need to re-add the presence

Reply:
Thank you for the notice, sadly - I think to prevent confusion, we need to rename the service name to match all our branding otherwise we know people will somehow get confused haha. Thank you Bas.

Reply:
Aight then you also have to change the folder name to the new one